deploy/kiva-ui.yml

```diff
@@ -18,7 +18,7 @@ spec:
       - name: kiva-ui
         image: kiva/ui:master
         command: [ "npm" ]
-        args: [ "start", "--", "--port=8888", "--config=k8sdev", "--prefix", "/ui/ui-build" ]
+        args: [ "start", "--", "--port=8888", "--config=k8sdev", "--prefix", "/ui" ]
```

Comment:
I'd recommend we delete `kiva-ui.yml` as it's not actually getting deployed by Helm (it lives outside the `charts` directory and otherwise isn't referenced anywhere I can see). If we want to do local k8s deployments we should set up our chart + values file to handle that scenario for us.